server/pbenchinacan/container-build.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+#
+# This script builds a container which, when run, starts a Pbench Server.
+#
+# Note:  successfully running this script required adding AUDIT_WRITE (for the
+#        operation of sshd) and CAP_SETFCAP (for the installation of httpd) to
+#        the list of default capabilities in /etc/containers/containers.conf
+#        (and, if that file doesn't exist, you'll need to create it with the
+#        other default capabilities, e.g., see
+#        https://man.archlinux.org/man/containers.conf.5.en#CONTAINERS_TABLE and
+#        https://github.com/containers/common/blob/da56e470c0c57c27e91bdc844b32c5dab6611394/pkg/config/containers.conf#L48)
+#
+
+set -o errexit
+
+# Locations inside the container
+INSTALL_ROOT=/opt/pbench-server
+SERVER_LIB=${INSTALL_ROOT}/lib
+SERVER_BIN=${INSTALL_ROOT}/bin
+CONF_PATH=${SERVER_LIB}/config/pbench-server.cfg
+HOSTNAME_F=pbenchinacan
+
+# Locations on the host
+GITTOP=${GITTOP:-$(git rev-parse --show-toplevel)}
+PBINC_DIR=${GITTOP}/server/pbenchinacan
+
+# Open a copy of the base container.  Docker format is required in order to set
+# the hostname.
+container=$(buildah from --format=docker quay.io/pbench/pbench-devel-rhel8:main)
+
+# We could mount the container filesystem and access it directly, but we
+# instead access it with buildah commands.
+# mnt=$(buildah mount $container)
+
+# Consider adding -v datavolume for the Server data, and perhaps SQL and ES data
+buildah config \
+    --label maintainer="Nick Dokos <ndokos@redhat.com>" \
```

Comment:
When did *that* happen?

Reply:
It was like that when you gave me the file....  😉